jscomp/others/js_re.ml

```diff
@@ -23,169 +23,146 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
 (**
-  Provides bindings for JavaScript Regular Expressions
-
-  {4 Syntax sugar}
-  ReScript provides a bit of syntax sugar for regex literals: `[%re "/foo/g"]`
```

Comment:
I guess this part was removed since there is already some coverage in the manual: https://rescript-lang.org/docs/manual/latest/primitive-types#regular-expression

I guess it would generally be better to document the details in the manual instead to make the information more visible?

Reply:
I see your point. Agree, the intro is better suited for the manual.

So, I’d just live here the note telling the API is not immutable.